### ISSUE TITLE
Combining this with ChatGLM

### ISSUE THREAD

**Issue by VarLad** [THUDM/CodeGeeX#123]
Is it possible to combine this project with the ChatGLM project such that we can have better completion, and can also have a conversation with a chat bot, to drive the AI to write better completion code?

**Comment by Stanislas0** [comment]
Yeah, this is possible and we have an early attempt to add Chat function in our newest extension, please have a try! 

VS Code Extension: https://marketplace.visualstudio.com/items?itemName=aminer.codegeex
Jetbrains Extension: https://plugins.jetbrains.com/plugin/20587-codegeex
